Sources/_Helpers/URLSession+AsyncAwait.swift

```diff
@@ -0,0 +1,43 @@
+#if canImport(FoundationNetworking)
+import Foundation
+import FoundationNetworking
+
+public enum URLSessionPolyfillError: Error {
+  case noDataNoErrorReturned
+}
+
+extension URLSession {
```

Comment:
Foundation on non-Darwin platforms doesn't have async/await support, so these are just shims for those APIs